examples/generate.sh

```diff
@@ -72,21 +69,27 @@ if [ $OVERWRITE -ne 1 ] && [ -d "$OUTPUT_DIR" ]; then
 fi
 
 mkdir -p "${OUTPUT_DIR}"
+kubectl create namespace "${NAMESPACE}"
+
+# Get enhanced envsubst version to evaluate expressions like ${VAR:=default}
```

Comment:
So this is what we were missing in generate.sh and that is why CAPI pod was failing not being able to evaluate some expressions, since we used older version of envsubst.